.changeset/ninety-avocados-act.md

```diff
@@ -0,0 +1,5 @@
+---
+'@khanacademy/graphql-flow': patch
+---
+
+Support custom scalars as variables in an operation
```

Comment:
Why is this file called `ninety-avocados-act`?

Reply:
the changeset has this funny way of naming files. I'm not really sure what the point is